File: python/model/ads_size.py
from dataclasses import dataclass


@dataclass(frozen=True)
class AdSize:
    """広告サイズクラス。広告のサイズIDに基づいて大きさを管理する"""

    size_id: int  # サイズID
    ads_width: int  # 広告幅
    ads_height: int  # 広告高さ

File: golang/model/ads_size.go
package model

// 広告案件に紐づく特徴量を保存するクラス。
type AdSize struct {
	SizeId    uint `db:"size_id"`    // 広告サイズID
	AdsWidth  uint `db:"ads_width"`  // 広告幅
	AdsHeight uint `db:"ads_height"` // 広告高さ
}
